src/libs/actions/Report.js

```diff
@@ -441,13 +441,14 @@ function unsubscribeFromReportChannel(reportID) {
  * set of participants and redirect to it.
  *
  * @param {String[]} participants
+ * @returns {Promise<Number>} report id
```

Comment:
For consistency let's mention the return variable as `reportID` instead of `report id`